### PR TITLE
vscode: fix extension icon

### PR DESCRIPTION
**Pull request by elena-pudovkina**
Add path to icon in package.json.in

### CODE REVIEW

File: extensions/vscode/package.json.in [hunk 1]
```diff
@@ -33,38 +34,36 @@
...
-						"description": "The path to the wake binary that is used to create the timeline. If invalid, the timeline functionality is not provided.",
-						"scope": "window",
-						"type": "string"
-					}
-				}
-			}
-		],
+		"configuration": {
+            "id": "wakeLanguageServer",
+            "title": "Wake",
```
Comment (ag-eitilt) [inline extensions/vscode/package.json.in:39]: This file seems to mainly use tabs, so much as I like spaces myself, they seem to misalign it.

Comment (elena-pudovkina) [reply extensions/vscode/package.json.in]: Arrays and lists are legal, the configuration didn't use to be an array though, which is why I changed it. I changed it back now

Comment (elena-pudovkina) [reply extensions/vscode/package.json.in]: The file is partly spaces and partly tabs. Should I rewrite it to use tabs only?

Comment (ag-eitilt) [reply extensions/vscode/package.json.in]: If it's already a mix, then let's not fold it into the extension fix.  If you wanted to standardize, I wouldn't complain, but it's probably not required, and should be its own PR if you wanted to do so.

Comment (ag-eitilt) [reply extensions/vscode/package.json.in]: Or if you've got the commit already ready, that works too! 😁

Comment (elena-pudovkina) [reply extensions/vscode/package.json.in]: I pushed the commit!

**Pull request merged**